news/migrations/0041_auto_20220517_1353.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.2.5 on 2022-05-17 18:53
+
+from django.db import migrations
+import wagtail.core.blocks
+import wagtail.core.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('news', '0040_alter_newsarticle_content_types'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
```

Comment:
probably alright this time but remember that Dante said it's best to remove references in one PR, then do the migration in the next release

Reply:
I'll remember next time